kafka-ui-react-app/src/components/common/Input/Input.tsx

```diff
@@ -35,10 +36,21 @@ const Input: React.FC<InputProps> = ({
           inputSize={inputSize}
           {...methods.register(name, { ...hookFormOptions })}
           hasLeftIcon={!!leftIcon}
+          onKeyDown={(e) => {
+            if (e.key === 'e' && type === 'number') {
```

Comment:
@workshur it's not letting to type letter 'e' but when copy pasting work containing letter 'e' it's not working, for that I can use e.keyCode or e.which but those two are deprecated, so should we keep this as it is or solve using deprecated methods?

Reply:
@rAzizbekyan  can we use pattern here? something like `pattern="^[-/d]/d*$"` for `integer only` input

Reply:
it's not solving the problem as we need to not let letter 'e' to be typed,but via pattern we let 'e' to be typed but got respective error for that